README.md

```diff
@@ -71,6 +71,7 @@ Arguments:
     --dry-run                    Verifies configuration and prints commands that
                                  would be executed, but does not do any work.
 
+    --skia                       Builds Flutter web apps with CanvasKit.
```

Comment:
UX thought: if we're talking about this is terms of `CanvisKit` (skia is an implementation detail) should we call the flag `--canvas-kit`?